datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -1077,3 +1077,17 @@ b 5
 c 4
 d 4
 e 4
+
+
+
+# array_agg_zero
```

Comment:
Writing these tests was amazing -- I just added the queries and the `--complete` mode added the expected results ❤️